internal/resource/config/oauth/authserversettings/oauth_auth_server_settings_resource.go

```diff
@@ -380,18 +382,24 @@ func (r *oauthAuthServerSettingsResource) Schema(ctx context.Context, req resour
 			},
 			"registered_authorization_path": schema.StringAttribute{
 				Description: "The Registered Authorization Path is concatenated to PingFederate base URL to generate 'verification_url' and 'verification_url_complete' values in a Device Authorization request. PingFederate listens to this path if specified",
-				Required:    true,
+				Optional:    true,
+				Computed:    true,
+				Default:     stringdefault.StaticString(""),
 				Validators: []validator.String{
 					configvalidators.StartsWith("/"),
 				},
 			},
 			"pending_authorization_timeout": schema.Int64Attribute{
 				Description: "The 'device_code' and 'user_code' timeout, in seconds.",
-				Required:    true,
+				Optional:    true,
+				Computed:    true,
+				Default:     int64default.StaticInt64(600),
```

Comment:
Where do these defaults come from?

Reply:
PingFederate GET request after spinning a new instance

Reply:
Also in the response when the value is not sent in.

Reply:
Ok thanks. I forgot this was a settings resource as well.